tests/unit/button/button_options.js

```diff
@@ -8,14 +8,14 @@ module( "button: options" );
 test( "disabled, explicit value", function() {
 	expect( 9 );
 
-	var element = $( "#radio01" ).button({ disabled: false });
+	var element = $( "#button" ).button({ disabled: false });
 	deepEqual( element.button( "option", "disabled" ), false, "disabled option set to false" );
```

Comment:
Let's delete this kind of assertion, checking an option immediately after init, it doesn't provide any value.